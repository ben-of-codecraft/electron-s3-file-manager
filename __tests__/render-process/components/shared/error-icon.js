const React = require('react');
const ReactTestRenderer = require('react-test-renderer');
const ErrorIcon = require('../../../../src/renderer-process/components/shared/error-icon');

describe('error-icon', () => {
	test('render', () => {
		const render = ReactTestRenderer.create(<ErrorIcon/>);

		expect(render.toJSON()).toMatchSnapshot();
	});

	test('render with props', () => {
		const render = ReactTestRenderer.create(<ErrorIcon className="m-2"/>);

		expect(render.toJSON()).toMatchSnapshot();
	});
});
