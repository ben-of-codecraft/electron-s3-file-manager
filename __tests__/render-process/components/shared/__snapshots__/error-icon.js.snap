// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error-icon render 1`] = `
<div
  className="swal2-icon swal2-error swal2-icon-show d-flex"
>
  <span
    className="swal2-x-mark"
  >
    <span
      className="swal2-x-mark-line-left"
    />
    <span
      className="swal2-x-mark-line-right"
    />
  </span>
</div>
`;

exports[`error-icon render with props 1`] = `
<div
  className="swal2-icon swal2-error swal2-icon-show d-flex m-2"
>
  <span
    className="swal2-x-mark"
  >
    <span
      className="swal2-x-mark-line-left"
    />
    <span
      className="swal2-x-mark-line-right"
    />
  </span>
</div>
`;
